wandb/sdk/launch/agent/agent.py

```diff
@@ -52,6 +57,12 @@ class JobAndRunStatus:
     def job_completed(self) -> bool:
         return self.completed or self.failed_to_start
 
+    def populate_run_info_from_launch_project(
```

Comment:
nit
```suggestion
    def update_run_info(
```

Reply:
we are passing in a type LaunchProject, easily inferred. 100000% up to you though.